etc/services/osf-localhost.json

```diff
@@ -3,7 +3,7 @@
   "id" : 203948234207230,
   "name" : "",
   "description" : "",
-  "serviceId" : "^https?://(localhost|127.0.0.1|dev.osf.io):500[0-9]/(?!oauth/callback).*",
+  "serviceId" : "^https?://(localhost|127.0.0.1|dev.osf.io):500[0-9]/(?!(oauth/callback|erpc|prereg|(login/?\\?next=https?(%3A|:)(%2F|/)(%2F|/)(localhost|127.0.0.1|dev.osf.io)(%3A|:)500[0-9](%2F|/)preprints(%2F|/)))).*",
```

Comment:
possible values (setup on staging2 currently for testing)

|service file|evaluationOrder|
| --- | --- |
|cas.json|100|
|oauth2.json|100|
|osf.json|2000|
|osf-campaigns-erpc.json|1000|
|osf-campaigns-prereg.json|1000|
|preprints-engrxiv.json|1000|
|preprints-osf.json|1000|
|preprints-psyarxiv.json|1000|
preprints-socarxiv.json|1000|